### DISCUSSION TITLE
Cross Cluster Chat

### DISCUSSION THREAD

**Discussion by furio85** [gameserverapp/Platform#290] [Suggestions]
It would be nice to get the option to allow cross cluster chat.  For example:  2 different clusters can see chat within the game.  Will help with in-game help issues or players just wanting to communicate!  Some Api's can do this, but only the paid versions.  Would be nice to include in an already paid subscription that has its own mod already running that is capable.  One less thing to pay and program!  Thanks for considering!

**Comment by mvaessen** [comment]
Thanks for your suggestion, this has been added!
